src/stories/Other/Breakdown.stories.js

```diff
@@ -30,11 +30,11 @@ export const Default = () => {
                     </Button>
                 </Breakdown.SubtotalItem>
 
-                <Breakdown.Item value={62} info={<CardIcon />}>
+                <Breakdown.Item value={62} info={"*0259"} methodIcon={<CardIcon />}>
```

Comment:
![Screenshot 2022-11-23 at 11 34 03](https://user-images.githubusercontent.com/65708/203481702-52f24023-f02c-434d-bbb9-876d44471425.png)

The design has the info in a while block, whereas this doesn't show it like that.